### DISCUSSION TITLE
9.1.2

### DISCUSSION THREAD

**Discussion by Geolim4** [TruCopilot/phpfastcache#870] [Releases]
- __API__
  - Upgraded Phpfastcache API to `4.2.0` ([see changes](https://github.com/PHPSocialNetwork/phpfastcache/blob/9.1.2/CHANGELOG_API.md))
- __Core__
  - Rewrote some core code to improve code maintainability & readability following Scrutinizer and Phpstan recommendations
  - Fixed an issue with tags not properly reinitialized when a backend driver returns an expired cache item
- __Drivers__
  - Fixed #866 // Deprecated Method Cassandra\ExecutionOptions starting of Cassandra 1.3
- __Misc__
  - Increased PHPSTAN level to 6
  - Fixed multiple fails of Travis CI
  - Migrated Github issue templates from Markdown to YAML configurations

<hr /><em>This discussion was created from the release <a href='https://github.com/PHPSocialNetwork/phpfastcache/releases/tag/9.1.2'>9.1.2</a>.</em>

**Comment by Awilum** [comment]
see changes - link is broken

**Reply by Geolim4** [reply]
It's directly due to how the discussion is auto-generated xD

See on release, link is working:
https://github.com/PHPSocialNetwork/phpfastcache/releases/tag/9.1.2
